### PR TITLE
feat: mocking data in schemas

### PR DESCRIPTION
**Pull request by andriy-ilin**

### CODE REVIEW

File: __schema__/capitationContracts.graphql [hunk 1]
```diff
@@ -89,9 +89,12 @@ type CapitationContract implements Node & Contract {
   "The ID of an object"
   id: ID!
   "Primary key identifier from the database"
-  databaseId: ID!
+  databaseId: UUID!
   "Human redable contract number"
   contractNumber: String
+    @fake(
+      randexp: "^[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}-[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}-[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}$"
```
Comment (danakonovalenko) [inline __schema__/capitationContracts.graphql]:
```suggestion
      randexp: "^\\d{4}-[\\dAEHKMPTX]{4}-[\\dAEHKMPTX]{4}$"
```

File: __schema__/reimbursementContractRequests.graphql [hunk 2]
```diff
@@ -96,9 +96,12 @@ type ReimbursementContractRequest implements Node & ContractRequest {
   "The ID of an object."
   id: ID!
   "Primary key identifier from the database."
-  databaseId: ID!
+  databaseId: UUID!
   "Human readable number of contract request."
   contractNumber: String
+    @fake(
+      randexp: "^[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}-[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}-[0-9A-Za-zА-ЯҐЇІЄа-яґїіє]{4}$"
```
Comment (danakonovalenko) [inline __schema__/reimbursementContractRequests.graphql]:
```suggestion
      randexp: "^\\d{4}-[\\dAEHKMPTX]{4}-[\\dAEHKMPTX]{4}$"
```

File: __schema__/contractRequests.graphql [hunk 3]
```diff
@@ -25,9 +25,9 @@ interface ContractRequest {
   "Content od printout form in html format."
   printoutContent: String
   "The day when contract takes effect."
-  startDate: Date! @fake(locale: "uk", method: "date.past")
+  startDate: Date!
   "The day when contract ends."
-  endDate: Date! @fake(locale: "uk", method: "date.past")
+  endDate: Date!
```
Comment (danakonovalenko) [inline __schema__/contractRequests.graphql]:
```suggestion
  endDate: Date! @fake(locale: "uk", method: "date.future", args: [10])
```

Comment (andriy-ilin) [reply __schema__/contractRequests.graphql]: in interface we don't need directive fake, we use it only on type

File: __schema__/scalars.graphql [hunk 4]
```diff
@@ -1,3 +1,13 @@
+"""
+The `ID` scalar type represents a  global id.
+"""
+scalar ID
+
+"""
+The `UUID` scalar type represents a id.
```
Comment (kiurchv) [inline __schema__/scalars.graphql]: We should point here that UUID is an internal ID value that stored in our DB

**Pull request merged**